### ISSUE TITLE
Automaticaly start/stop time counting

### ISSUE THREAD

**Issue by dskibarko** [speedy32129/time_logger#42]
Plugin will be better if automaticaly start/stop time counting were present if task status transient detected.
This very actuality for Redmine there Agile board used. Time start/stop will be from agile. 
For example describe main idea:
1. If  Task status chаnged to "working", then timer start counting time for this task.
2. If  Task status chаnged from "working" to over(should be administrator configured), then timer stop counting time for this task.
3. If  "Task N" status chаnged to "working" and for over task "Task M" time is counting, then 
    - "Task M" automaticaly moved from "working" to "pause" status(should be administrator configured)
    - Stop time counting for "Task M" (because only one job can be in progress).

**Comment by speedy32129** [comment]
I'm currently in development of a branch that is adding multiple loggers or timers (optionally configured by administrator).  

The next phase is multiple transitions.  So if a developer starts a task the status will update from open to working then when the developer stops working on it it will go back to the previous status open.

If I get your drift your asking for status change based on when someone manually changes the status to something different. you want to automatically end the existing logger and start a new one?

**Comment by dskibarko** [comment]
Yes, main idea -> time counting for task if task in state "working".
Some people was used JIRA (on previous company) and JIRA time logging is baised on current task state "working".
Multiplle transition (MP-method described by you) is good solution to. 
May be  should be better to have both methods (you MP-method and JIRA-method)???

**Comment by dskibarko** [comment]
Hi!  Have you a progress or not?

**Comment by dskibarko** [comment]
Hello,
what about good songs feature  "The next phase is multiple transitions. So if a developer starts a task the status will update from open to working then when the developer stops working on it it will go back to the previous status open."???
Is it in progress?
